static/compiler.js

```diff
@@ -1399,6 +1381,32 @@ Compiler.prototype.updateLibsDropdown = function () {
                 if (libsList.children().length > 0)
                     libsList.append($('<hr>').addClass('lib-separator'));
 
+                var onChecked = _.bind(function (e) {
```

Comment:
I want this function back on top so there's only one version. It's here now because the node tree was a bit confusing at 3am to select the correct parent element.

Maybe https://stackoverflow.com/a/9467172/4410614 can help